x-pack/plugins/cases/public/components/user_actions/comment/comment.test.tsx

```diff
@@ -156,6 +205,55 @@ describe('createCommentUserActionBuilder', () => {
 
       expect(screen.getByText('removed attachment')).toBeInTheDocument();
     });
+
+    it('renders correctly when deleting a persistable state attachment with getAttachmentRemovalObject defined', async () => {
+      const persistableStateAttachmentTypeRegistry = new PersistableStateAttachmentTypeRegistry();
+      const attachment = getPersistableStateAttachment();
+      persistableStateAttachmentTypeRegistry.register({
+        ...attachment,
+        getAttachmentRemovalObject: () => ({
```

Comment:
Could you also test that `getAttachmentRemovalObject` is getting called with the expected `props`?

Reply:
Thanks for that!  I found a bug.